src/commands/moderation/reason.ts

```diff
@@ -20,6 +20,11 @@ export default class extends Command {
             }));
         args.shift();
 
+        if (message.guild.me?.permissions.has('MANAGE_MESSAGES'))
+            return message.error(message.translate('common:INSUFFICIENT_PERMISSIONS', {
+                permission: 'MANAGE_MESSAGES'
+            }));
+
```

Comment:
This permission isn't required for the command to run, only to delete the messages. I don't think we need to require manage message permission for this command.